torch_ort_inference/torch_ort/ortinferencemodule/__init__.py

```diff
@@ -33,7 +33,6 @@ def _defined_from_envvar(name, default_value, warn=True):
 # NOTE: To *change* values in runtime, import onnxruntime.training.ortmodule and
 # assign them new values. Importing them directly do not propagate changes.
 ################################################################################
-ONNX_OPSET_VERSION = 14
```

Comment:
Does the same opset version from ORTModule also work for ORTInferenceModule?

Reply:
MSFT mentioned that the latest opset_version used by torch.onnx during export should be used since it is for inference.